customcommands/assets/customcommands-editcmd.html

```diff
@@ -740,6 +740,146 @@ <h2 class="card-title">Custom Command Information</h2>
     });
 </script>
 
+<link rel="stylesheet" href="/static/vendorr/codemirror/lib/codemirror.css">
+<link rel="stylesheet" href="/static/vendorr/codemirror/theme/ayu-mirage.css">
+<script src="/static/vendorr/codemirror/lib/codemirror.js"></script>
+<script src="/static/vendorr/codemirror/mode/markdown/markdown.js"></script>
+<script src="/static/vendorr/codemirror/addon/mode/multiplex.js"></script>
+<script src="/static/vendorr/codemirror/addon/display/placeholder.js"></script>
+<style>
+    .CodeMirror {
+        width: 100% !important;
+        max-width: 95% !important;
```

Comment:
```diff
+       resize: vertical;
```
This allows resizing the editor like the standard textarea did previously.

Reply:
Added, @galen8183